test/JDBC/upgrade/15_4/schedule

```diff
@@ -0,0 +1,450 @@
+# Schedule File for JDBC Test Framework for local run
```

Comment:
We would need 14.10/schedule as well, which should be replica of latest/schedule from 14.latest

Reply:
Done